server/profile.js

```diff
@@ -1,16 +1,37 @@
 const { Pool, Client } = require('pg')
 
 const pool = new Pool({
-  user: process.env.user,
-  host: process.env.host,
-  database: process.env.database,
-  password: process.env.password,
-  port: process.env.port,
+    user: process.env.user,
+    host: process.env.host,
+    database: process.env.database,
+    password: process.env.password,
+    port: process.env.port,
 });
 
+// Application States
+const READY_FOR_VEFIFICATION = 'Application Pending Employer Validation';
+const VERIFIED = 'Application Completed Employer Validation';
```

Comment:
So in addition to Christina's comment saying that `Application` here is redundant, we should also have another informational state that tells us if more information is required that'll be triggered when clicking the `More Information Required` button.

Reply:
I agree - if we decide to do this, we will need to make the changes on the DB first and then update the enum values on the server. I cannot change the string as the Database will throw an error due to the mismatch or error types. So there is a dependency on the DB first. 
If we would like to make this change as part of this ticket we will need to update our ACs as the current ticket responsible for making sure all the table column are set up is ticket 3 (GET VIEW UPDATE)

Reply:
Update: After collaborating with out DB SE (@christinajiyunlee) we were able to implement this feedback